metrics.sh

```diff
@@ -24,25 +24,31 @@ function getMetrics () {
     echo "Getting Dependencies amounts metrics..."
     getDependenciesAmounts
 
-    echo "Getting Launch Time metric..."
-    node ${metrics_folder}/launch_time.js 
+    if [ ! -z "$FIREBASE_PROJECT_ID" ] ; then
+      echo "Getting Launch Time metric..."
+      node ${metrics_folder}/launch_time.js 
+    fi
 }
 
 function copyCredentials () {
-  if [ "$BITRISE_GIT_BRANCH" == "$PRODUCTION_BRANCH" ] ; then
-    echo ${PRODUCTION_CREDENTIAL_GOOGLE_JSON} | base64 --decode > ${metrics_folder}/credential.json
-    echo "Copy: PRODUCTION_CREDENTIAL_GOOGLE_JSON"
-  else
-    if [ "$BITRISE_GIT_BRANCH" == "$STAGING_BRANCH" ] ; then
-      echo ${STAGING_CREDENTIAL_GOOGLE_JSON} | base64 --decode > ${metrics_folder}/credential.json
-      echo "Copy: STAGING_CREDENTIAL_GOOGLE_JSON"
+  if [ -z "$FIREBASE_PROJECT_ID" ] ; then
+    echo "Project do not have Firebase"
```

Comment:
![tenor (3)](https://user-images.githubusercontent.com/7256996/75905447-a3037300-5e23-11ea-8dc5-983a2d261ebd.gif)

```suggestion
    echo "Project does not have Firebase"
```